test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -300,5 +300,83 @@ Describe "Get-Help should find pattern help files" -Tags "CI" {
        $help.Category | Should BeExactly "Alias"
        $help.Synopsis | Should BeExactly "Where-Object"
     }
+}
+
+Describe "help function uses full view on non-Windows" -Tags "CI" {
+    It "help should return full view without -Full switch" -Skip:($IsWindows) {
+        $gpsHelp = (help Microsoft.PowerShell.Management\Get-Process)
+        $gpsHelp | Where-Object {$_ -cmatch '^PARAMETERS'} | Should Not BeNullOrEmpty
+    }
+
+    It "help should return full view even with -Full switch" -Skip:($IsWindows) {
+        $gpsHelp = (help Microsoft.PowerShell.Management\Get-Process -Full)
+        $gpsHelp | Where-Object {$_ -cmatch '^PARAMETERS'} | Should Not BeNullOrEmpty
+    }
+
+    It "help should not append -Full when not using AllUsersView parameter set" -Skip:($IsWindows) {
+        $gpsHelp = (help Microsoft.PowerShell.Management\Get-Process -Parameter Name)
+        $gpsHelp | Where-Object {$_ -cmatch '^PARAMETERS'} | Should BeNullOrEmpty
+    }
+}
+
+Describe "help function does not use full view on Windows by default" -Tags "CI" {
+    BeforeAll {
+        $origPager = $null
+        if (Test-Path env:PAGER) {
+            $origPager = $env:PAGER
+            Remove-Item Env:\PAGER
+        }
+    }
+
+    AfterAll {
+        if ($origPager) {
+            New-Item Env:\PAGER -Value $origPager > $null
+        }
+    }
+
+    It "help should not return full view without -Full switch" -Skip:(!$IsWindows) {
```

Comment:
Since you already have and after all the `Skip` should be done over there instead of repeating it in every `It`.

https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk